### DISCUSSION TITLE
Getting this error message when I open the Intro to github workflow learning path, "We are unable to access the starter code for this assignment. Please contact your Classroom admin."

### DISCUSSION THREAD

**Discussion by MustafaMulla29** [community/community#140365] [GitHub Education]
### Hi everyone,

![image](https://github.com/user-attachments/assets/e2ac31b4-19c7-4338-aa3f-30fb41950e72)

This is the message I am getting. Help would appreciated!

**Comment by vinod820** [comment]
I am also facing the same issue

**Comment by tausif-fardin** [comment]
Any update on this issue ?

**Reply by MustafaMulla29** [reply]
No updates, still the same issue!
